Webpack-dev-server 3.1.12 seems broken

* Operating System: Mac osx high Sierra
* Node Version: v 6.9.4
* NPM Version: 3.10.10
* webpack Version: 4.20.2
* webpack-dev-server Version: 3.1.12

- [x ] This is a **bug**
- [ ] This is a **modification** request

### Code

<!--
  If you have a large amount of code to share which demonstrates the problem
  you're experiencing, or your webpack config is very large, please provide a link
  to your repository rather than pasting code. We'd also encourage you to use a
  Github Gist link instead of pasting code. Otherwise, please paste relevant
  short snippets below.

  For bugs, please do consider providing a link to a stripped-down, bare-bones
  repo that can reproduce the problem you're experiencing. Many times, bugs
  aren't actual bugs, but rather specific issues with loaders, plugins, or
  an environment/OS. Problems with complicated or large applications will almost
  always require this to be triaged.
-->

```js
  // webpack.config.js
devServer: {
    contentBase: join(__dirname, 'dist')
  }
```

### Expected Behavior
webpack dev server should start a local dev server

### Actual Behavior
Crashes with the following error:
```
node_modules/webpack-dev-server/lib/Server.js:654
  ).hostname;
  ^
SyntaxError: Unexpected token )
    at Object.exports.runInThisContext (vm.js:76:16)
    at Module._compile (module.js:542:28)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Applications/MAMP/htdocs/node-ts/frontend/node_modules/webpack-dev-server/bin/webpack-dev-server.js:38:16)
    at Module._compile (module.js:570:32)

npm ERR! Darwin 17.4.0
npm ERR! argv "/Users/admin/.nvm/versions/node/v6.9.4/bin/node" "/Users/admin/.nvm/versions/node/v6.9.4/bin/npm" "run" "wds"
npm ERR! node v6.9.4
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE
npm ERR! js-design-pattern@1.0.0 wds: `webpack-dev-server --progress --colors`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the js-design-pattern@1.0.0 wds script 'webpack-dev-server --progress --colors'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the js-design-pattern package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     webpack-dev-server --progress --colors
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs js-design-pattern
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls js-design-pattern
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Applications/MAMP/htdocs/node-ts/frontend/npm-debug.log
```
### For Bugs; How can we reproduce the behavior?
Installing webpack-dev-server@3.1.12 (same node version might be required)

--- a/lib/Server.js
+++ b/lib/Server.js
@@ -650,7 +650,7 @@ Server.prototype.checkHost = function (headers, headerToCheck) {
     // if hostHeader doesn't have scheme, add // for parsing.
     /^(.+:)?\/\//.test(hostHeader) ? hostHeader : `//${hostHeader}`,
     false,
-    true,
+    true
   ).hostname;
   // always allow requests with explicit IPv4 or IPv6-address.
   // A note on IPv6 addresses:

